xtable-core/src/test/java/org/apache/xtable/delta/TestDeltaValueConverter.java

```diff
@@ -82,6 +82,15 @@ void parseWrongDateTime() throws ParseException {
     assertThrows(ParseException.class, () -> strictDateFormat.parse(wrongDateTime));
   }
 
+  @ParameterizedTest
+  @MethodSource("nonNumericValuesForColStats")
+  public void formattedDifferentNonNumericValuesFromDeltaColumnStat(
```

Comment:
nitpick: `public` can be removed

Reply:
For info, I use `public` by following these tests: https://github.com/apache/incubator-xtable/blob/main/xtable-core/src/test/java/org/apache/xtable/delta/TestDeltaValueConverter.java#L44
but happy to remove all of them

Reply:
We have a linter that flags this stuff which is why I selfishly flagged this for my own internal cherry-pick later :) I am fine with just doing this on new tests for now. Maybe later we can integrate a linter into the the ASF toolkit as well

Reply:
Sure. Removed.